LayoutTests/svg/crash-svg-null-maskimage.html

```diff
@@ -0,0 +1,6 @@
+<svg style="mask-image: none, url()"></svg>
```

Comment:
Maybe try wrapping the whole test is a `<body></body>` ?

I see that LayoutTests/svg/transforms/svg-transform-box-initial.html is able to run on glib ports and does something similar.

Reply:
Revised test case per above suggestion. Thanks